docs/topics/vscode-extension-reviewing-projects.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-reviewing-projects_{context}']
+
+= Reviewing and resolving issues with the {ProductName} extension
+
+You can review and resolve issues identified by the {ProductName} extension in the Issue Run configuration pane.
+
+== Reviewing issues
```

Comment:
This structure is too granular. Remove the "Reviewing and resolving issues" header and make "Reviewing issues" (change to "Reviewing migration issues) as level 1. There is no point in creating all these subheaders if they do not add anything.